tox.ini

```diff
@@ -0,0 +1,30 @@
+[tox]
+envlist = py37
+
+[testenv]
+setenv =
+    PYTHONDONTWRITEBYTECODE=1
+
+passenv =
+    HOME
+
+deps =
+    coverage>=4.2b1
```

Comment:
Nit: do we still want the 4.2b1? I think we're successfully using the 4.2 in beetmoverscript and potentially others. https://github.com/mozilla-releng/beetmoverscript/blob/master/tox.ini#L16

Reply:
Good call. I removed this condition.